pandas/core/reshape/reshape.py

```diff
@@ -628,7 +628,11 @@ def _convert_level_number(level_num, columns):
     levsize = len(level_labels)
     drop_cols = []
     for key in unique_groups:
-        loc = this.columns.get_loc(key)
+        try:
```

Comment:
what test exercises this?

Reply:
https://github.com/pandas-dev/pandas/blob/35b2aba7386619dc156ca245cab781d3aea37789/pandas/tests/test_multilevel.py#L1195
https://github.com/pandas-dev/pandas/blob/35b2aba7386619dc156ca245cab781d3aea37789/pandas/tests/frame/test_reshape.py#L136
https://github.com/pandas-dev/pandas/blob/35b2aba7386619dc156ca245cab781d3aea37789/pandas/tests/frame/test_reshape.py#L730

These were the 3. And it looks like Travis doesn't have permalinks to specific lines like GitHub?

Reply:
Also just took a look at the test that caused most of the builds to fail:
https://github.com/pandas-dev/pandas/blob/35b2aba7386619dc156ca245cab781d3aea37789/pandas/tests/frame/test_mutate_columns.py#L186-L198
and we just changed this so I'll go ahead and negate the `assert`.